doc/open-extensions/muc_light.md

```diff
@@ -992,6 +1011,8 @@ The notifications contain a list of items. The item list may be different from t
 </message>
 ```
 
+`hag88` has been kicked off the room and therefore gets only their own affiliation change of type 'none'.
```

Comment:
`hag88` has been kicked out of the room and therefore gets only their own affiliation change of type 'none'.